news/2021-04-06-mappers-report-march.md

```diff
@@ -0,0 +1,173 @@
+| ![QAT/NAT Badge (3 years)](https://assets.ppy.sh/profile-badges/QAT3y.jpg "Longstanding contribution to the QAT/NAT - 3 years") | [-Mo-](https://osu.ppy.sh/users/2202163), [Lasse](https://osu.ppy.sh/users/896613), [Naxess](https://osu.ppy.sh/users/8129817) |
+| ![QAT/NAT Badge (4 years)](https://assets.ppy.sh/profile-badges/QAT4y.jpg "Longstanding contribution to the QAT/NAT - 4 years") | [Feerum](https://osu.ppy.sh/users/4815717), [pishifat](https://osu.ppy.sh/users/3178418) |
+
+## <a id="mappers-showcase"></a>Mappers' Showcase
+
+It's time for the Mappers' Showcase. This month, we will feature two hidden gems from osu!taiko. Should you be into hitting drums, you've got to check them out for sure!
+
+Huge thanks to the selectors of this issue, [Hivie](https://osu.ppy.sh/users/14102976) and [radar](https://osu.ppy.sh/users/7131099)!
+
+### Maimaing
+
+A newer yet very promising mapper, [Maimaing](https://osu.ppy.sh/users/14520910) has quickly caught the ropes of mapping and has been on the rise the moment he started delving into the editor since late 2020.
+
+His name quickly started spreading among the Japanese community and later into the general scene upon his first steps towards the ranked section. His very first ranked beatmap, [PSYQUI - Hype feat. Such (lapix Remix)](https://osu.ppy.sh/beatmapsets/1316175), a well-crafted map featuring fascinating rhythms and patterning, provides the players with a great experience that gives the song justice.
+
+Another event that highlights his ever-growing potential is him joining the roster of the custom mappers for osu!taiko World Cup 2021! He has attained the chance to create a DT map for the Group Stage. Thus, he took the opportunity to bring the v-tuber atmosphere to the TWC in the form of [hololive IDOL PROJECT - SHIJOSHUGI ADTRUCK](https://osu.ppy.sh/beatmapsets/1387197), offering a challenging and fun experience for players to compete on, featuring rhythms that will surely put your finger control skills to the test.
+
+While he still has a long way to reach his full potential, it will be quite fascinating to see how Maimaing will develop as a mapper later on. He's always trying something new, so make sure to check out his other works, and maybe subscribe to this profile if you're interested enough in what he'll make next.
+
+### X a v y
+
+A mapper, whose popularity has been on the rise among high-level players, [X a v y](https://osu.ppy.sh/users/3738344) burst into the ranked section for the first time in early 2021 with [Houshou Marine - Ahoy!! Warera Houshou Kaizokudan\*](https://osu.ppy.sh/beatmapsets/1282787), a marathon of a song composed by a fairly popular Virtual YouTuber. However, this didn't happen before amassing a very sizable catalog of pending/graveyarded challenge maps. If you are a high-level player, he is sure to have something in his graveyard that will suit your tastes!
```

Comment:
```suggestion
A mapper whose popularity has been on the rise among high-level players, [X a v y](https://osu.ppy.sh/users/3738344) burst into the ranked section for the first time in early 2021 with [Houshou Marine - Ahoy!! Warera Houshou Kaizokudan\*](https://osu.ppy.sh/beatmapsets/1282787), a marathon version of a pirate v-tuber's debut song. 

Though this might've been his first foray into ranked, X a v y's beatmap graveyard is all but crammed full of a massive catalogue of challenging maps, so make sure to give it a look if you fancy yourself up to the occasion!
```